web-marketplace/src/components/organisms/RegistryLayout/RegistryLayout.Header.tsx

```diff
@@ -108,7 +110,7 @@ const RegistryLayoutHeader: React.FC = () => {
         pathname={pathname}
         extras={
           <Box display="flex" justifyContent="center" alignItems="center">
-            <ListProject />
+            {clientConfig.listProject && <ListProject />}
```

Comment:
eventually we'll probably have a totally different header for terrasos, but this makes sense to do this temporarily to make the current header work without the wallet related providers